### ISSUE TITLE
Print widget not working

### ISSUE THREAD

**Issue by nathankota** [agrc/deq-enviro#580]
- Search layers: Air Monitoring by Station; Oil and Gas Wells; GWPZ; Water Rights POD
- Advanced Search: Tooele County
- Click print widget
- Click Export multiple times and the errors stack up within the Exported files section of the widget:

![image](https://github.com/agrc/deq-enviro/assets/15659642/26613bfc-10ae-42b2-98a2-10c9cdafd05b)

**Comment by stdavis** [comment]
This _might_ be a duplicate of #560...

**Comment by stdavis** [comment]
The mapserv print service is failing if you attempt to print after performing a search with the following error message in the JS: 

```
"print:cim-symbol-unsupported","CIMSymbol is not supported by a print service published from ArcMap"
```

When I attempted to publish a new export web map gp service to mapserv from pro, I got the following validation errors:

![Image](https://github.com/agrc/deq-enviro/assets/1326248/a582db1a-32bf-4c3b-b9f4-fc72e10ed4cb)

So I decided to just switch to our default AGOL print service which seems to work fine. The tradeoff is that the layout is not custom like the current app. I don't know if this will be a big deal or not to the client. If they want the custom layout, they will have to wait until we [upgrade mapserv](https://github.com/agrc/planning-queue/issues/166).
